packages/server/src/users/api/UserDto.ts

```diff
@@ -0,0 +1,53 @@
+import { ApiProperty, ApiPropertyOptional, OmitType } from '@nestjs/swagger';
+
+import { FavouriteRestaurantDto } from '../../restaurants/api/RestaurantDto';
+import { FavouriteDishDto } from '../../restaurants/dishes/api/DishDto';
+import { ApiObjectIdProperty } from '../../shared';
+
+interface Card {
+  number: number;
+  expirationDate: Date;
+  securityCode: number;
+}
+
+class UserDto {
+  @ApiObjectIdProperty()
+  readonly id: string;
+
+  @ApiPropertyOptional()
+  readonly name: string;
+
+  @ApiPropertyOptional()
+  readonly surname: string;
+
+  @ApiProperty()
+  readonly email: string;
+
+  @ApiProperty()
+  readonly password: string;
```

Comment:
Możesz zrobić drugi typ DTO, bez hasła, bo o ile przy rejestracji czy coś hasło może być wysyłane do serwera, to raczej nie będzie nigdy sytuacji, że serwer zwróci nam z danymi użytkownika (zhashowane) hasło.

Żeby wszystkiego nie przepisywać od początku możesz skorzystać z rzeczy w stylu `PickType`, albo w tym przypadku bardziej `OmitType` (przykłady w `ArticleDto.ts`/`RestaurantDto.ts`).